grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -1345,4 +1345,25 @@ class ProductService {
             importProducts(products)
         }
     }
+
+    ProductCatalogItem createOrUpdateProductCatalogItem(Map params) {
```

Comment:
Move this to a new ProductCatalogService.

Reply:
done